modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -382,6 +382,146 @@
 #define NRFX_WDT1_ENABLED 1
 #endif
 
+#ifdef CONFIG_LOG_NRFX_ADC
+#define NRFX_ADC_CONFIG_LOG_ENABLED 1
+#endif
```

Comment:
Why are those translations located separately? It feels natural to have them together with other ones related to a given driver, for example:
```
#ifdef CONFIG_NRFX_EGU
#define NRFX_EGU_ENABLED 1
#endif
#ifdef CONFIG_NRFX_EGU_LOG
#define NRFX_EGU_CONFIG_LOG_ENABLED 1
#endif
#ifdef CONFIG_NRFX_EGU0
#define NRFX_EGU0_ENABLED 1
#endif
#ifdef CONFIG_NRFX_EGU1
#define NRFX_EGU1_ENABLED 1
#endif
...
```